sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -3072,3 +3072,34 @@ opentitan_functest(
         "//sw/device/silicon_creator/lib/drivers:rstmgr",
     ],
 )
+
+############################################################
+# Another big comment here
+# to prevent git merge from interleaving the tests
+# TODO: remove this guard comment
+
+opentitan_functest(
+    name = "rom_e2e_ret_ram_keep",
+    srcs = ["rom_e2e_ret_ram_keep_test.c"],
+    cw310 = cw310_params(
+        bitstream = "//hw/bitstream:rom",
+        tags = ["vivado"] + maybe_skip_in_ci(CONST.LCV.RMA),
```

Comment:
This test technically doesn't need this tag since it uses the unaltered ROM bitstream, but I've added this tag to keep it grouped with the other tests. If we want to start running it in CI, I can remove this.

Reply:
Thanks for adding this tag, it will be easier to review when we clean up this file.